db/migrate/20230222155424_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.datetime :updatedAt
+      t.datetime :createdAt
```

Comment:
- [ ] Kindly delete these columns, these are not required because 👉🏽 `t.timestamps` will crate updated_at and created_at columns